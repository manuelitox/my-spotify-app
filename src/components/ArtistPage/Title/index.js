import React from 'react'
import PropTypes from 'prop-types'

import StyledTitle from './styles'

const Title = ({ children }) => <StyledTitle>{ children }</StyledTitle>

Title.propTypes = {
  children: PropTypes.string.isRequired
}

export default Title