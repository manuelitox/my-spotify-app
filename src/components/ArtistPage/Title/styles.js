import styled from 'styled-components'
import PropTypes from 'prop-types'

const StyledTitle = styled.h2`
  color: ${ props => props.color };
  font-size: 2rem;
  font-weight: 500;
  margin: 0 0 40px;
  text-decoration: underline;
  @media only screen and (max-width: ${ props => props.theme.size.small }) {
    margin-bottom: 30px;
  }
`

StyledTitle.defaultProps = {
  color: 'white'
}

StyledTitle.propTypes = {
  color: PropTypes.string
}

export default StyledTitle