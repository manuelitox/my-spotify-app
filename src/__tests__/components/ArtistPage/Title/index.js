import React from 'react'
import { shallow } from 'enzyme'

import Title from 'components/ArtistPage/Title/'

describe('Component | Title', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <Title>Albums</Title>
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})