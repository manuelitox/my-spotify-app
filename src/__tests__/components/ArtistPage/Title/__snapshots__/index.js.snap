// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | Title should render a component 1`] = `
<styled.h2
  color="white"
>
  Albums
</styled.h2>
`;
